core/arch/arm/plat-imx/conf.mk

```
@@ -112,7 +113,7 @@ endif
 ifneq (,$(filter $(PLATFORM_FLAVOR),mx6qsabrelite mx6dlsabrelite))
 CFG_DDR_SIZE ?= 0x40000000
 CFG_NS_ENTRY_ADDR ?= 0x12000000
-CFG_UART_BASE ?= UART1_BASE
+CFG_UART_BASE ?= UART2_BASE
```

Unintended change of the SabreLite OP-TEE debug UART. See commit https://github.com/ms-iot/optee_os/commit/96e9aef4c4bf5c86e662e36897f113e6869bac3e
```suggestion
CFG_UART_BASE ?= UART1_BASE
```

---

Fixed.